x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/batch_actions.tsx

```diff
@@ -39,17 +40,18 @@ export const getBatchItems = ({
   hasMlPermissions,
   loadingRuleIds,
   reFetchRules,
+  refetchPrePackagedRulesStatus,
   rules,
   selectedRuleIds,
   hasActionsPrivileges,
 }: GetBatchItems) => {
-  const selectedRules = selectedRuleIds.reduce((acc, id) => {
+  const selectedRules = selectedRuleIds.reduce<Record<string, Rule>>((acc, id) => {
     const found = rules.find((r) => r.id === id);
     if (found != null) {
       return { [id]: found, ...acc };
     }
     return acc;
-  }, {} as Record<string, Rule>);
```

Comment:
🎉 yay for one less `as`

Reply:
I couldn't resist when I was in there and just happened by it. Templates are so much better.